cmd/antrea-agent/agent.go

```diff
@@ -431,6 +431,15 @@ func run(o *Options) error {
 	// if AntreaPolicy feature is enabled.
 	statusManagerEnabled := antreaPolicyEnabled
 	loggingEnabled := antreaPolicyEnabled
+	var auditLoggerOptions *networkpolicy.AntreaPolicyLoggerOptions
+	if loggingEnabled {
```

Comment:
Good point, maybe it should still be `AntreaLogger`? Just to confirm, I remember K8s NetworkPolicy log in Antrea was introduced to be shown along side AntreaPolicy, so it needs both _AntreaPolicy enabled_ and _logging enabled in the Namespace_, if `loggingEnabled` is always true, this might be a different criteria?

Reply:
Good point. I don't think there is a strong reason to couple audit logging and the `AntreaPolicy` feature gate. One should be able to enable logging for K8s NPs for a Namespace, even when the `AntreaPolicy` feature gate is disabled.

I chose the struct name `AntreaPolicyLoggerOptions` because the audit logging implementation uses a struct named `AntreaPolicyLogger` but I agree that it could / should be something more generic (e.g., `AuditLogger` and `AuditLoggerOptions`).

I'll open an issue to follow up on this.